packages/components/src/components/cookies-consent.tsx

```diff
@@ -45,7 +47,16 @@ export function CookiesConsent(props: CookiesConsentProps) {
         >
           Privacy Policy
         </a>
-        <CallToAction variant="tertiary" onClick={onAccept} className="px-4 py-2">
+        <CallToAction
+          variant="tertiary"
+          onClick={() => {
+            setConsented('closing');
+            if (typeof localStorage !== 'undefined') {
+              localStorage.setItem('cookies', 'true');
+            }
```

Comment:
why this wrap? didn't get

> [Wrap localStorage access in typeof check for server components](https://github.com/the-guild-org/docs/pull/1844/commits/f5711b92727cb3c831128cec8e16cbcad4b9052b)

Reply:
I was getting an error during server components render without it. No idea _why_, but wrapping it in a typeof check helped.